vars/neoDeploy.groovy

```diff
@@ -27,14 +27,17 @@ def call(parameters = [:]) {
         'warAction'
         ]
 
+    List generalConfigurationKeys = [
+        'neoHome'
+        ]
+
     List stepConfigurationKeys = [
         'account',
         'dockerEnvVars',
         'dockerImage',
         'dockerOptions',
         'host',
-        'neoCredentialsId',
-        'neoHome'
```

Comment:
Regadless if we have it in addition in the general section. Shouldn't it be possible to configure neoHome at the level of the step?

Reply:
Yes, I created a new PR #84 to be able to do this.

Reply:
I had only a short look into PR #84. This deal apparently with shifting the order of the config layers. But if ```neoHome``` is removed from the ```stepConfigurationKeys``` it is not possible to configure ```neoHome``` at the level of the step configuration. Regardless which order of the config layers we have.